<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: lightpink</string>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleTXT">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Client客户端界面</string>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="cmdLineSendServerEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="showRevServer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="RevServerTXT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>接收服务器信息：</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="showSituation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>281</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="SendserverTXT">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入要发送至服务器信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="SituationTXT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>状态及输出：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>484</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="ServerIpTXT">
      <property name="text">
       <string>连接服务器IP：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cmdLineServerIPEdit">
      <property name="styleSheet">
       <string notr="true">background-color: white;
</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="portTXT">
      <property name="text">
       <string>端口号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cmdLinePortEdit">
      <property name="styleSheet">
       <string notr="true">background-color:white;
</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="connectButton">
      <property name="styleSheet">
       <string notr="true">background-color: white;
</string>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>188</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="IDcmdLineEdit">
      <property name="styleSheet">
       <string notr="true">background-color:white;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
